[DOCS-15275] Add OP Traces for Preview (#38665)
DataDog/documentation: hugo/content/en/observability_pipelines/configuration/set_up_pipelines.md

[6]: /observability_pipelines/configuration/pipeline_simulation/
[11]: /observability_pipelines/search_syntax/metrics/

{{% /tab %}}
{{% tab "Traces" %}}

<div class="alert alert-info">Traces for Observability Pipelines is in Preview. Contact your account manager to request access.</div>

1. Navigate to [Observability Pipelines][1].
1. Select the [Trace Sampling][2] template, or click {{< ui >}}New Pipeline{{< /ui >}} and select {{< ui >}}Traces Pipeline{{< /ui >}}.
1. Select and set up a [trace source][3].
1. Select and set up [destinations][5] for your processed traces.
1. Click {{< ui >}}Edit{{< /ui >}} on a processor group to add, update, and validate your [processors][4] using [Pipeline Simulation][6].
    - **Notes**:
      - For a pipeline canvas, there is a limit of 25 processor groups and a total of 150 processors.

>       - If you want to copy a processor, click the copy icon for that processor and then paste it (`Cmd+V` on Mac, `Ctrl+V` on Windows/Linux).

Reviewer: ```suggestion
      - To copy a processor, click the copy icon for that processor and then paste it (`Cmd+V` on Mac, `Ctrl+V` on Windows or Linux).
```

Removing the slash + I think it reads ok with just "to copy"!
